weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsCharts.tsx

```diff
@@ -182,7 +182,7 @@ export const CallsCharts = ({
   return (
     <Tailwind>
       {/* setting the width to the width of the screen minus the sidebar width because of overflow: 'hidden' properties in SimplePageLayout causing issues */}
-      <div className="md:w-[calc(100vw-56px)]">
+      <div className="w-full">
```

Comment:
We still need the condition for `md:w-[calc(100vw-56px)]` otherwise the padding is off (see the comment above this)

Beta link (padding is off on the right on widths >= the breakpoint of 768px

see:

https://github.com/user-attachments/assets/cfcd9bdb-3ad5-4e23-a71f-b1baef03c406

vs correct padding:

https://github.com/user-attachments/assets/12841246-9401-4890-993e-d92e42bf277c

Reply:
Okay I'm now setting to `w-full` but using width - 56px for a max width